cognite_toolkit/cdf_tk/load.py

```diff
@@ -728,6 +752,30 @@ def load_file(self, filepath: Path, ToolGlobals: CDFToolConfig) -> FileMetadata
             )
         except Exception:
             files = FileMetadataList.load(load_yaml_inject_variables(filepath, ToolGlobals.environment_variables()))
+        if len(files.data) == 1:
+            # If we have a file with exact one file config, check to see if this is a pattern to expand
+            if "$FILENAME" in files.data[0].external_id or "":
```

Comment:
Have we documented this pattern?

Reply:
Will do in the yaml reference that is next on my list.